### PR TITLE
 :tada: Client side initial

### PR DESCRIPTION
**Pull request by SaraOchayon**
basic react app

### CODE REVIEW

**Comment by NomiMagnus** [comment]
> basic react app

Change to Add

**Comment by NomiMagnus** [comment]
Remove unneccesary files.

File: client/README.md
Comment (NomiMagnus) [inline client/README.md:1]: We'll have to change...

File: client/public/favicon.ico
Comment (NomiMagnus) [inline client/public/favicon.ico:1]: Delete this file.

File: client/public/index.html
Comment (NomiMagnus) [inline client/public/index.html:1]: Change this file.
Must be empty now.

File: client/public/logo192.png
Comment (NomiMagnus) [inline client/public/logo192.png:1]: Remove.

File: client/public/logo512.png
Comment (NomiMagnus) [inline client/public/logo512.png:1]: Remove.

File: client/public/manifest.json
Comment (NomiMagnus) [inline client/public/manifest.json:1]: Check if important!

File: client/public/robots.txt
Comment (NomiMagnus) [inline client/public/robots.txt:1]: ???

File: client/public/manifest.json
Comment (NomiMagnus) [reply client/public/manifest.json:1]: Delete!

File: client/src/App.css
Comment (NomiMagnus) [inline client/src/App.css:1]: Clear content!

File: client/src/App.js
Comment (NomiMagnus) [inline client/src/App.js:1]: Clear contant!

File: client/src/index.css
Comment (NomiMagnus) [inline client/src/index.css:1]: Clear content!

File: client/README.md
Comment (SaraOchayon) [reply client/README.md:1]: Shouldn't we wait for the future and then write in a more detailed and knowledgeable way?

File: client/src/logo.svg
Comment (NomiMagnus) [inline client/src/logo.svg:1]: Delete!

File: client/public/manifest.json
Comment (SaraOchayon) [reply client/public/manifest.json:1]: not important!!

**Review by NomiMagnus — COMMENTED** [review]
Wonderful job, keep going!

File: client/README.md
Comment (NomiMagnus) [reply client/README.md:1]: 👍 Thats why I wrote ...

File: client/public/logo192.png
Comment (NomiMagnus) [reply client/public/logo192.png:1]: ???

File: client/README.md
Comment (SaraOchayon) [reply client/README.md:1]: ok

File: client/public/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+   
+    <title>React Side</title>
```
Comment (NomiMagnus) [inline client/public/index.html:7]: Change to 'ScheduleMaster'

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (NomiMagnus) [inline client/public/index.html:5]: Havo to change icon.
Somewhen in the future...

File: client/src/App.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+import './App.css';
+
+function App() {
+  return (
+   <h1>welcome</h1>
```
Comment (NomiMagnus) [inline client/src/App.js:6]: remove

**Review by NomiMagnus — COMMENTED** [review]
Well done!

**Comment by NomiMagnus** [comment]
Pay attention to branch name I've changed!

**Review by NomiMagnus — APPROVED** [review]
Perfect!

**Pull request merged**